### PR TITLE
refactor: PRT - Replace cookbook/specs with lava-config specs

### PR DESCRIPTION
**Pull request by shleikes**

### CODE REVIEW

**Review by oren-lava — CHANGES_REQUESTED** [review]

File: utils/keeper/spec.go [hunk 1]
```diff
@@ -144,19 +144,55 @@ func GetASpec(specIndex, getToTopMostPath string, ctxArg *sdk.Context, keeper *k
...
+
+	proposalDirectories := []string{
+		"specs/mainnet-1/specs/",
+		"specs/testnet-2/specs/",
 	}
-	for _, fileName := range proposalFiles {
-		spec, err := GetSpecFromPath(getToTopMostPath+proposalDirectory+fileName, specIndex, &ctx, keeper)
-		if err == nil {
-			return spec, nil
+	baseProposalFiles := []string{
```
[the remark this answers is not in the document]
Comment (shleikes) [reply utils/keeper/spec.go:152]: You're right about ethermint, but why Solana?

File: scripts/useful_commands.sh [hunk 2]
```diff
@@ -168,4 +168,44 @@ done
 echo "---------------------------------------------"
 echo "-            ENV Validation Done            -"
 echo "---------------------------------------------"
+}
+
+get_base_specs() {
+    local priority_specs=(
```
[the remark this answers is not in the document]
Comment (shleikes) [reply scripts/useful_commands.sh:174]: Why?

File: specs/mainnet-1/specs/blast.json [hunk 3]
```diff
@@ -0,0 +1,114 @@
+{
```
[the remark this answers is not in the document]
Comment (shleikes) [reply specs/mainnet-1/specs/blast.json:1]: Will change now, thanks!
I think I'll put the spec that are not deployed anywhere, back into `cookbook/specs`, what do you think?

File: x/spec/keeper/spec_test.go [hunk 4]
```diff
@@ -815,21 +815,28 @@ func TestApiCollectionsExpandAndInheritance(t *testing.T) {
 	}
 }
 
-func TestCookbookSpecs(t *testing.T) {
+func TestMainnetSpecs(t *testing.T) {
```
[the remark this answers is not in the document]
Comment (shleikes) [reply x/spec/keeper/spec_test.go]: Fixed in 379a2d1a152d3e43bdd986f75cec47f8d48379fe

File: utils/keeper/spec.go (hunk 1 above)
[the remark this answers is not in the document]
Comment (oren-lava) [reply utils/keeper/spec.go:152]: I'm thought that big chains that other specs import are in this category and koii imports solana. Might not be right, so it's not mandatory

File: specs/mainnet-1/specs/blast.json (hunk 3 above)
[the remark this answers is not in the document]
Comment (oren-lava) [reply specs/mainnet-1/specs/blast.json:1]: Maybe `cookbook/not_deployed`? or add a readme that states:
1. `mainnet` specs are specs that are on both mainnet and testnet
2. `testnet` specs are only on testnet
3. `specs` specs are not deployed

[the remark this answers is not in the document]
Comment (shleikes) [reply specs/mainnet-1/specs/blast.json]: Fixed

**Review by Yaroms — APPROVED** [review]

**Pull request merged**